types/node/path.d.ts

```diff
@@ -123,7 +123,7 @@ declare module 'path' {
              *
              * @param p the path to evaluate.
              */
-            extname(p: string): string;
+            extname(p: string): '' | `.${string}`;
```

Comment:
I am not sure how that'd work with runtime strings, also  I don't think we can use string interpolation types on the version supported by node dt.

Reply:
>  I don't think we can use string interpolation types on the version supported by node dt

Yeah, you're right!